contracts/src/contracts/Teleporter/AvalancheICTTRouter.sol

```diff
@@ -39,15 +41,23 @@ contract AvalancheICTTRouter is Ownable, ReentrancyGuard, IAvalancheICTTRouter {
      * @notice Destination chain ID => token address => DestinationBridge
      * @notice Address `0x0` is used for the native token
      */
-    mapping(
-        bytes32 destinationChainID => mapping(address token => DestinationBridge destinationBridge)
-    ) public tokenDestinationChainToDestinationBridge;
+    mapping(bytes32 destinationChainID => mapping(address token => DestinationBridge destinationBridge))
```

Comment:
I believe we said that we'd change the order of the mapping to `token => destinationChain` to be consistent.